MVP Module/Source/Feed UI/Controllers/FeedViewController.swift

```
@@ -22,6 +23,7 @@ public final class FeedViewController: UITableViewController, UITableViewDataSou
 	}
 
 	@IBAction private func refresh() {
+		errorView?.hideMessage()
```

The Presenter should decide when to hide/show errors. See my other comment.

For example, in the `didStartLoadingFeed()` method, the Presenter can call  `errorView.display(FeedErrorViewModel(message: .none))` to hide the error.

This way, the ViewController doesn't need to decide when to hide the error. It's easier to manage presentation state transitions when all changes are centralized in the presenter.